etc/signal-desktop.profile

```diff
@@ -0,0 +1,35 @@
+# Firejail profile for sinal-desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/signal-desktop.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+noblacklist ~/.config/Signal
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-programs.inc
+
+mkdir ~/.config/Signal
+whitelist ${DOWNLOADS}
+whitelist ~/.config/Signal
+include /etc/firejail/whitelist-common.inc
+
```

Comment:
If you're on 0.9.51 can you test if `include /etc/firejail/whitelist-var-common.inc` works?

Reply:
Seems to work fine